libHVCAENx527App/src/HVCAENx527.c

```diff
@@ -1283,6 +1273,8 @@ int
 CAENx527SetAllChParVal( HVCRATE *cr, char *pname, void *val)
 {
 	int i, j;
+	int rval;
+	rval = 0;
 	int nset;
```

Comment:
mixing assign and declaration will fail on earlier c compilers like vs2010, i will fix